apps/dotcms-ui/src/app/api/services/guards/can-deactivate-guard.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
```

Comment:
We need to rename this guard to something more specific related to the feature where is used `FeatureSpecificCanActivateGuard`... in this case since is related to layout editor: `LayoutEditorCanDeactivateGuard`.